torch/export/exported_program.py

```diff
@@ -605,4 +605,8 @@ def get_shape_env(gm):
         for k, v in shape_env.var_to_range.items()
```

Comment:
@angelayi this looks better?
```suggestion
        for k, v in shape_env.runtime_var_to_range.items()
```

Reply:
Oh, the issue is that runtime_var_to_range only contains ranges for unbacked symints, not input symints.

Reply:
oh ok.